{{cookiecutter.project_slug}}/README.md

```diff
@@ -1,5 +1,43 @@
 # {{ cookiecutter.project_name }}
 
+# Description
+
 {{ cookiecutter.project_short_description }}
 
 Created by {{ cookiecutter.full_name }} ({{ cookiecutter.email }})
+
+# Development Environment
+
+This project is designed to run on [Neuro Platform](https://neu.ro), so you can jump into problem-solving right away.
+
+## Directory Structure
+
+| Mount Point                                  | Description           | Storage URI                                                                  |
+|:-------------------------------------------- |:--------------------- |:---------------------------------------------------------------------------- |
+|`/project/data/`                              | Data                  | `storage:{{ cookiecutter.project_slug }}/data/`                              |
+|`/project/{{ cookiecutter.code_directory }}/` | Python modules        | `storage:{{ cookiecutter.project_slug }}/{{ cookiecutter.code_directory }}/` |
+|`/project/notebooks/`                         | Jupyter notebooks     | `storage:{{ cookiecutter.project_slug }}/notebooks/`                         |
+|`/project/results/`                           | Logs and results      | `storage:{{ cookiecutter.project_slug }}/results/`                           |
+
+## Development
+
+Follow the instructions below to setup the environment and start Jupyter development session.
+
+## Neuro Platform
+
+* Setup development environment `make setup`
+* Run Jupyter with GPU: `make jupyter`
+* Kill Jupyter: `make kill-jupyter`
+* Get list of available template commands: `make help`
+
+# Data
+
+## Uploading via Web UI
+
+From local machine run `make filebrowser` and open job's URL from your mobile device or desktop. 
```

Comment:
*On* local machine

Reply:
same, open URL *on* the mobile device

Reply:
If `on local machine`, then `on mobile device`, without `the`